maven-embedder/src/main/java/org/apache/maven/cli/MavenCli.java

```diff
@@ -447,6 +443,10 @@ private void informativeCommands(CliRequest cliRequest) throws ExitException {
             }
             throw new ExitException(0);
         }
+
+        if (cliRequest.rootDirectory == null) {
```

Comment:
since i'm using alpha8-snapshot i'm tempted to setit in debug, wdyt?
this is not a maven requirement (and must not be) so I wouldnt log at info or more level until it is used (so maybe warn in the interpolation and debug there?)